README.rst

```diff
@@ -616,7 +617,10 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``tokens`` (``array``): list of tokens to fix; defaults to ``['extra']``
+  - ``tokens`` (a subset of ``['break', 'continue', 'extra', 'return', 'throw',
+    'use', 'useTrait', 'use_trait', 'curly_brace_block',
```

Comment:
is there an easy way to not list deprecated options (like `useTrait` here) or is this something that can only be done on a higher/later version?

Reply:
I would not hide deprecated options/rules.
Imagine that one is using them (because they were not deprecated few released ago), and now s/he is looking for docs for it...